packages/bot/src/commands/reply.ts

```diff
@@ -57,34 +55,6 @@ export default class implements Command<ApplicationCommandType.ChatInput> {
 	}
 
 	public async handle(interaction: ChatInputCommandInteraction<'cached'>) {
```

Comment:
```suggestion
	public handle(interaction: ChatInputCommandInteraction<'cached'>) {
```

Also remove the constructor from this class, GH UI won't let me suggest that.